clients/csharp-dotnet-core/voice-assistant-test/README.md

```diff
@@ -148,7 +151,7 @@ Here is the full list:
 >`string | required | "0"`. A unique value within the test file that identifies this dialog. You can identify a dialog by giving each one a random GUID value, an integer counter, or anything else. Intended to be short.
 
 >#### Description
->`string | optional | null | "Dialog for reserving airline ticket"`. Free-form text description of what this dialog does, to help you remember. Does not have to be unique.
+>`string | optional | null | "Dialog for reserving airline ticket"`. Free-form text description of what this dialog does, to help you remember. Does not have to be unique. 
 

```

Comment:
I think this new feature deserves a new section in the README's Topic chapter. A paragraph that talks about the need to run the same test twice, once with text input and another time with audio. If there are test failures associate with audio input (e.g. speech recognition not returning  the expected result), then the test with audio input will fail but the same test with text input will pass, which is important information to have when trying to root cause the failures.